git-tutorial.markdown

```diff
@@ -98,11 +98,77 @@ If you have your old installation under `~/alice` then we suggest you use a
 different directory to avoid confusion.
 
 
+### I don't use aliBuild
+
+If you do not use aliBuild you can keep using your previous installation
+commands without any modification. You need however to clone the ALICE
+repositories from GitHub:
+
+```bash
+cd ~/alice
+git clone https://github.com/alisw/AliRoot
+git clone https://github.com/alisw/AliPhysics
+```
+
+
+### I have participated to the GitHub test
+
+> These instructions are only valid from the time the GitHub migration has been
+> completed!
+
+If you have participated to the GitHub test and you want to sync with the
+current official AliRoot/AliPhysics and alidist repositories from upstream,
+starting from the day when Git migration is effective you can follow this
+procedure and you will save some build time.
+
+Assuming you have your GitHub test installation under `~/alice`, update your
+recipes:
+
+```bash
+cd ~/alice
+rm -rf alidist/
+aliBuild init
+```
+
+The `aliBuild init` command will re-download the production version of the
+build recipes. Now move into your AliRoot and AliPhysics directories and update
+them. Note that the following operation is **destructive** and you will lose
+any local change for good!
+
+```bash
+cd ~/alice/AliPhysics
```

Comment:
Given that I don't understand why suggesting to `pull` alidist (which is tiny) and remove completely AliRoot/AliPhysics (which are quite fat), I might agree that everything can boil down to:

```
rm -rf alidist AliRoot AliPhysics
aliBuild init AliRoot,AliPhysics
```

I can change it to that (which would also work on `lxplus` where `git fetch --all` doesn't) :-(